examples/Components/forcefield/RestShapeSpringsForceField2.scn

```diff
@@ -23,7 +23,7 @@
 
     <Node name="Object2">
         <EulerImplicitSolver rayleighMass="0" rayleighStiffness="0"/>
-        <SparseLUSolver name="LULinearSolver"/>
+        <SparseLUSolver template="CompressedRowSparseMatrixMat3x3d" name="LULinearSolver"/>
```

Comment:
I am surprised `SparseLUSolver` is used here.
I am also surprised that you chose `CompressedRowSparseMatrixMat3x3d`, whereas the DoF type is `Rigid3`. Did you benchmark `CompressedRowSparseMatrixMat3x3d` and `CompressedRowSparseMatrixd`?

Reply:
Without the template we obtain the following error:

```
[WARNING] [SparseLUSolver(LULinearSolver)] Template '' incorrect, used CompressedRowSparseMatrixMat3x3d in the list:
	* CompressedRowSparseMatrixMat3x3d
	* CompressedRowSparseMatrixd
```
I didn't push the study deeper for the performances, but they almost converged simultaneously. But if you have a preferred template let me know.

Reply:
Usually, if there is at least one rigid in the system, CompressedRowSparseMatrixd should be prefered as a first guess. But profiling says which is best. It's only a matter of performances. For an example, I would go with CompressedRowSparseMatrixd